meson_options.txt

```diff
@@ -208,7 +208,7 @@ option(
 option(
     'ibm-management-console',
     type: 'feature',
-    value: 'disabled',
+    value: 'enabled',
```

Comment:
why enable this here and not in the bbappend ?

Reply:
right, removed this as its already enabled in bbappend.